backend-opal/src/server/stockServer.js

```diff
@@ -0,0 +1,8 @@
+const dbDriver = require('./db/driver');
```

Comment:
@Yuan-EPAM can you try requiring `dbDriver` in another file too, if the server starts?

Reply:
also this is not a stockServer, this is a stockService or portfolioService